scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/model/GoalId.java

```diff
@@ -1,4 +1,4 @@
-package gov.nasa.jpl.aerie.scheduler.server.models;
+package gov.nasa.jpl.aerie.scheduler.model;
 
 import java.util.Optional;
 
```

Comment:
Nonblocking because this could be split into its own work, but I'm bringing this up because your changes need to interact with this design choice: Why is `goalInvocationId` `Optional`? The only time it is not defined outside of tests is for `refreshSchedulingProcedureParameterTypes`, and more specifically for `SpecificationRepository::updateGoalParameterSchema`. My gut preference is to just continue threading `goalId`, `revision` down into this function rather than making a field that is **always** otherwise present `Optional`

Reply:
If the invocation is deleted in the database, an activity can end up where the goal id is non-null and the invocation id is null. So since the `GoalId` record is used in other places I figured we should represent that.